modules/migration-rsync-manually-overriding.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+:_content-type: PROCEDURE
+[id="migration-rsync-manually-overriding_{context}"]
+= Manually overriding default non-root operation for data transfer
+
+Although running Rsync pods as non-root user works in most cases, data transfer might fail when when you run workloads as root user on the source side. {mtc-short} provides two ways to manually override default non-root operation for data transfer:
+
+* Configure all migrations to run an Rsync pod as root on the destination cluster for all migrations.
+* Run an Rsync pod as root on the destination cluster per migration.
+
+In both cases, you must set the following labels on the source side of any namespaces that are running workloads with higher privileges prior to migration: `enforce`, `audit`, and `warn.`
```

Comment:
why do you need to set audit and warn labels?  they don't actually prevent/allow anything.

Reply:
also "set them" is meaningless without saying what they need to be set to.  So this should say "must set the following labels to `privileged`" (but again, only the enforce label really matters)

Reply:
@pranavgaikwad ^

Reply:
Just setting them also won't help you if you're running in a namespace that has these auto-labelled.